src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-size: 16px;
+  padding-top: 20px;
+  margin-top: 0;
+  background-color: #fff;
```

Comment:
I guess it's redundant (default value)

Reply:
fix it https://github.com/mate-academy/layout_search-bar-airbnb/pull/2977/files#r1120377108